src/core/Akka/Configuration/IHoconConfig.cs

```diff
@@ -0,0 +1,222 @@
+﻿// //-----------------------------------------------------------------------
+// // <copyright file="IHoconConfig.cs" company="Akka.NET Project">
+// //     Copyright (C) 2009-2020 Lightbend Inc. <http://www.lightbend.com>
+// //     Copyright (C) 2013-2020 .NET Foundation <https://github.com/akkadotnet/akka.net>
+// // </copyright>
+// //-----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+
+namespace Hocon
+{
+    public interface IHoconConfig
```

Comment:
The interface - doesn't expose any underlying HOCON object graph implementation.